### PR TITLE
feat: adding pt-br translation for managing resources page

### PR DESCRIPTION
**Pull request by ThiagoGMF**
# Changes

- Brazilian portuguese translation for the [managing resources](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/) page

# Related issues 

#13939

### CODE REVIEW

**Comment by tamarabarbosa** [comment]
sorry... I tagged this PR in mistake.

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+project/k8s/development
+├── configmap
+│   └── my-configmap.yaml
+├── deployment
+│   └── my-deployment.yaml
+└── pvc
+    └── my-pvc.yaml
+```
+
+Por padrão, executar uma operação em massa em `project/k8s/development` irá parar no primeiro nível do diretório, não processando nenhum subdiretório. Se tivéssemos tentado criar os recursos neste diretório usando o seguinte comando, teríamos encrotado um erro:
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:118]:
```suggestion
Por padrão, executar uma operação em massa em `project/k8s/development` irá parar no primeiro nível do diretório, não processando nenhum subdiretório. Se tivéssemos tentado criar os recursos neste diretório usando o seguinte comando, teríamos encrotado um erro:
```
```suggestion
Por padrão, executar uma operação em massa em `project/k8s/development` irá parar no primeiro nível do diretório, não processando nenhum subdiretório. Se tivéssemos tentado criar os recursos neste diretório usando o seguinte comando, teríamos encontrado um erro:
```

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+
+```shell
+kubectl apply -f project/k8s/development
+```
+
+```shell
+error: you must provide one or more resources by argument or filename (.json|.yaml|.yml|stdin)
+```
+
+Em invés disso, utilize a _flag_ `--recursive` ou `-R` com a _flag_ `--filename,-f` assim:
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:128]:
```suggestion
Ao invés disso, utilize a _flag_ `--recursive` ou `-R` com a _flag_ `--filename,-f` assim:
```

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+
+Por exemplo, diferentes aplicações podem usar diferentes valores para a _label_ `app`, mas uma aplicação multi-nível, como [_guestbook example_](https://github.com/kubernetes/examples/tree/master/guestbook/), também precisaria distinguir cada _tier_. O _frontend_ pode carregar as seguintes _labels_:
+
+```yaml
+     labels:
+        app: guestbook
+        tier: frontend
+```
+
+Enquanto o Redis _master_ e _slave_ teriam _labels_ de diferentes `tier`, e talvez até uma label adicional para `role`:
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:171]: Maybe you can change the words `master` and `slave` to others synonymous, we aren't not using these words anymore at the docs.

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+```shell
+horizontalpodautoscaler.autoscaling/my-nginx autoscaled
+```
+
+Agora as suas réplicas do nginx vão aumentar e diminuir conforme necessário, automaticamente.
+
+Para mais informações, por favor veja os documentos [kubectl _scale_](/docs/reference/generated/kubectl/kubectl-commands/#scale), [kubectl _autoscale_](/docs/reference/generated/kubectl/kubectl-commands/#autoscale) e [horizontal _pod autoscaler_](/docs/tasks/run-application/horizontal-pod-autoscale/).
+
+
+## Atualização _in-place_ de recursos
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:352]: "in-place" can be translated to "Em execução" or something like that, what do you think?

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hi @ThiagoGMF I left some comments below :)

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md (hunk 1 above)
Comment (ThiagoGMF) [reply content/pt-br/docs/concepts/cluster-administration/manage-deployment.md]: done

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md (hunk 2 above)
Comment (ThiagoGMF) [reply content/pt-br/docs/concepts/cluster-administration/manage-deployment.md]: done

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md (hunk 3 above)
Comment (ThiagoGMF) [reply content/pt-br/docs/concepts/cluster-administration/manage-deployment.md]: i changed to "primário" and "secundário" using this [link](https://en.wikipedia.org/wiki/Master/slave_(technology)#Possible_replacement_conventions) as a reference

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md (hunk 4 above)
Comment (ThiagoGMF) [reply content/pt-br/docs/concepts/cluster-administration/manage-deployment.md]: done

**Comment by edsoncelio** [comment]
Thanks!

/lgtm

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+
+Para mais informações, por favor veja os documentos [kubectl _scale_](/docs/reference/generated/kubectl/kubectl-commands/#scale), [kubectl _autoscale_](/docs/reference/generated/kubectl/kubectl-commands/#autoscale) e [horizontal _pod autoscaler_](/docs/tasks/run-application/horizontal-pod-autoscale/).
+
+## Atualização de recursos em execução
+
+Às vezes, é necessário fazer atualizações restritas e sem interrupções nos recursos que você criou.
+
+### kubectl _apply_
+
+É sugerido manter um conjunto de arquivos de configuração no control de origem
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:360]: Suggestion: source control in this sentence alludes to source control management (SCM) systems. In this sentence, "sistema de controle de versão de código-fonte" might read better than "control de origem".

**Review by stormqueen1990 — COMMENTED** [review]
Thanks for your contribution! 🎉 

I left some suggestions below. Another two overall suggestions that apply to the whole page:
- I think it's not necessary to have italic formatting in `kubectl` commands.
- There are some instances of "deployed" that could be replaced by "instalado", since they don't refer to the Kubernetes resource.

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 6]
```diff
@@ -0,0 +1,460 @@
...
+kubectl get $(kubectl create -f docs/concepts/cluster-administration/nginx/ -o name | grep service)
+kubectl create -f docs/concepts/cluster-administration/nginx/ -o name | grep service | xargs -i kubectl get {}
+```
+
+```shell
+NAME           TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)      AGE
+my-nginx-svc   LoadBalancer   10.0.0.208   <pending>     80/TCP       0s
+```
+
+Com os comandos acima, primeiro criamos recursos em `examples/application/nginx/exibimos os recursos criados com `-o name`com o formato de saída (imprimindo cada recurso no formato _resource/name_). Em seguida filtramos apenas o recurso _Service_, utilizando a ferramenta `grep`, e então o imprimimos com `kubectl get`.
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:100]: Minor typo: this is showing up with the regular text as preformatted. It is missing a backtick after `nginx/`.

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 7]
```diff
@@ -0,0 +1,460 @@
...
+
+```shell
+kubectl scale deployment my-nginx --current-replicas=1 --replicas=3
+```
+
+```
+deployment.apps/my-nginx scaled
+```
+
+Para atualizar para a versão 1.16.1, altere `.spec.template.spec.containers[0].image` do `nginx:1.14.2` para `nginx:1.16.1 usando os comandos kubectl anteriores.
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:449]: Another backtick missing here after `nginx:1.16.1`.

**Review by stormqueen1990 — COMMENTED** [review]
Found a few minor issues. Other than that, LGTM!

File: content/pt-br/docs/concepts/cluster-administration/manage-deployment.md [hunk 8]
```diff
@@ -0,0 +1,461 @@
...
+```
+
+```shell
+NAME                           READY     STATUS    RESTARTS   AGE       APP         TIER       ROLE
+guestbook-fe-4nlpb             1/1       Running   0          1m        guestbook   frontend   <none>
+guestbook-fe-ght6d             1/1       Running   0          1m        guestbook   frontend   <none>
+guestbook-fe-jpy62             1/1       Running   0          1m        guestbook   frontend   <none>
+guestbook-redis-master-5pg3b   1/1       Running   0          1m        guestbook   backend    master
+guestbook-redis-slave-2q2yf    1/1       Running   0          1m        guestbook   backend    slave
+guestbook-redis-slave-qgazl    1/1       Running   0          1m        guestbook   backend    slave
```
[the remark this answers is not in the document]
Comment (edsoncelio) [reply content/pt-br/docs/concepts/cluster-administration/manage-deployment.md:201]: Hmm, this is just a piece of code/output right? I don't know If makes sense update this, maybe we can see at the original content.

**Comment by divya-mohan0209** [comment]
Hello [ThiagoGMF](https://github.com/ThiagoGMF), would you be comfortable addressing the suggestions by @stormqueen1990 ?

**Comment by divya-mohan0209** [comment]
Hello @ThiagoGMF , I am closing this PR since there has been no response from your end. Please reopen the PR once you're comfortable working on the suggestions by @stormqueen1990 .
/close

**Pull request closed without merging**